src/extension.ts

```diff
@@ -46,6 +47,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 
 	applyUserSettings();
 
+	// call didact to register tutorial
+	await registerTutorialWithDidact(context);
```

Comment:
It doesn't need to be unregistered when Camel K is deactivated?

Reply:
All tutorials are unregistered and re-registered when the workbench starts, so no need to unregister. I am relying on extension activation to register any applicable tutorials for the view.

Reply:
it is possible to disable an extension, i think that in this case, the tutorial will still be registered even though it won't be available anymore.